presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -160,6 +161,22 @@ public DockerContainer withHealthCheck(Path healthCheckScript)
                 .withCreateContainerCmdModifier(command -> command.withHealthcheck(cmd));
     }
 
+    public DockerContainer withHealthCheckCommand(String command)
+    {
+        try {
+            File tempFile = File.createTempFile("health-check", ".sh");
+            tempFile.deleteOnExit();
+            tempFile.setExecutable(true);
+
+            Path scriptPath = Files.writeString(tempFile.toPath(), format("#!/usr/bin/env bash\n%s", command));
+            return withHealthCheck(scriptPath);
```

Comment:
At the end of the day this ends up as `.withTest(ImmutableList.of("CMD", "health.sh"))`.
So you have three unnecessary layers of indirection:

- temp file locally
- temp file in the container at fixed location (/usr/local/bin/health.sh)
- bash

all of this seems redundant of you accept command as `String... command` instead